src/Calf/ParMetalanguage.agda

```diff
@@ -34,9 +38,58 @@ bind/& : ∀ {A₁ A₂} {X} {v₁ v₂ f} (p₁ p₂ : ℂ) →
   bind {Σ++ A₁ λ _ → A₂} X (step (F A₁) p₁ (ret v₁) & step (F A₂) p₂ (ret v₂)) f ≡ step X (p₁ ⊗ p₂) (f (v₁ , v₂))
 bind/& _ _ = refl
 
+open import Calf.Types.Unit
+
 bound/par : {A₁ A₂ : tp pos} {e₁ : cmp (F A₁)} {e₂ : cmp (F A₂)} {c₁ c₂ : ℂ} →
   IsBounded A₁ e₁ c₁ →
   IsBounded A₂ e₂ c₂ →
   IsBounded (Σ++ A₁ λ _ → A₂) (e₁ & e₂) (c₁ ⊗ c₂)
-bound/par (⇓ a₁ withCost p₁' [ h-bounded₁ , h-≡₁ ]) (⇓ a₂ withCost p₂' [ h-bounded₂ , h-≡₂ ]) with eq/ref h-≡₁ | eq/ref h-≡₂
-... | refl | refl = ⇓ (a₁ , a₂) withCost (p₁' ⊗ p₂') [ (λ u → ⊗-mono-≤ (h-bounded₁ u) (h-bounded₂ u)) , (ret (eq/intro refl)) ]
+bound/par = {!   !}
```

Comment:
What assumptions are needed to prove this? It suffices to show any of:
- `(bind (F A₁) e₁ ret) & (bind (F A₂) e₂ ret)` ≲ `step (F unit) c₁ (ret triv) & step (F unit) c₂ (ret triv)` 
- `bind (F unit) (e₁ & e₂) (λ _ → result)` ≲ `bind (F unit) ((bind (F unit) e₁ λ _ → ret triv) & (bind (F unit) e₂ λ _ → ret triv)) (λ _ → result)`
- `bind (F unit) (e₁ & e₂) (λ _ → result)` ≲ `bind (F unit) (e₁ & e₂) (λ _ → bind (F unit) (ret triv & ret triv) (λ _ → result))`